inc/sairouterinterface.h

```diff
@@ -285,6 +285,17 @@ typedef enum _sai_router_interface_attr_t
      */
     SAI_ROUTER_INTERFACE_ATTR_DECREMENT_TTL,
 
+    /**
+     * @brief To flag that this interface is present locally
+     *
+     * In VOQ systems it is possible that the interface is remote
+     *
+     * @type bool
+     * @flags CREATE_ONLY
+     * @default true
+     */
+    SAI_ROUTER_INTERFACE_ATTR_IS_LOCAL,
+
```

Comment:
Currently a router interface has a type attribute defined as port, vlan, sub_port etc., it seems we need to add a system_port for the type, as none of the existing router_interface_type is correct when the router interface is pointing to a system port.